packages/manager/core/generator/app/templates/src/components/layout-helpers/Listing/DatagridWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { OsdsDatagrid } from '@ovhcloud/ods-component-datagrid/react';
+import { OsdsLink } from '@ovhcloud/ods-component-link/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import appConfig from '@/{{appName}}.config';
+import ConfigInterface from '@/configInterface';
+import { ReactFormatter } from '@ovh-ux/manager-core-utils';
+
+function LinkService({ cellData, navigate }: any) {
+  return (
+    <OsdsLink
+      color={ODS_THEME_COLOR_INTENT.primary}
+      onClick={() => navigate(`/${cellData}`)}
```

Comment:
Pass onClick action here to avoid passing navigate

Reply:
```suggestion
      onClick={() => onClick({ cellData })}
```